### PR TITLE
feat: Added support for setting Megabits per second threshold in cloudflare_notification_policy  

### LINKED ISSUE #2384: cloudflare_notification_policy does not support Megabit per Second filters
### Confirmation

- [X] My issue isn't already found on the issue tracker.
- [X] I have replicated my issue using the latest version of the provider and it is still present.

### Terraform and Cloudflare provider version

Terraform v1.3.9
on darwin_arm64
+ provider registry.terraform.io/cloudflare/cloudflare v3.34.0

### Affected resource(s)

- cloudflare_notification_policy

### Terraform configuration files

```hcl
resource "cloudflare_notification_policy" "advanced_ddos_attack_l4_alert" {
  account_id  = local.account_id
  name        = "Advanced Layer 3/4 DDoS Attack Alert"
  enabled     = true
  alert_type  = "advanced_ddos_attack_l4_alert"

  filters {
    megabits_per_second = ["50"]
    packets_per_second = ["50000"]
  }
}
```

### Link to debug output

N/A

### Panic output

_No response_

### Expected output

Resource is managed properly

### Actual output

```
│ Error: failed to set filters: Invalid address to set: []string{"filters", "0", "megabits_per_second"}
```

### Steps to reproduce

1. Create resource has exemplified above

### Additional factoids

_No response_

### References

_No response_

### PR DESCRIPTION
**Pull request by atsushii**
Added support for setting Megabits per second threshold in cloudflare_notification_policy 

Closes #2384

### CODE REVIEW

**Review by jacobbednarz — CHANGES_REQUESTED** [review]
thanks for the PR! we will also need to include updates for the documentation (https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/contributing/registry-documentation.md) which should be as straight forward as `make docs` but YMMV.

File: docs/resources/pages_project.md [hunk 1]
```diff
@@ -174,7 +174,7 @@ Optional:
 - `fail_open` (Boolean) Fail open used for Pages Functions. Defaults to `false`.
 - `kv_namespaces` (Map of String) KV namespaces used for Pages Functions.
 - `r2_buckets` (Map of String) R2 Buckets used for Pages Functions.
-- `secrets` (Map of String) Encrypted environment variables for Pages Functions.
+- `secrets` (Map of String, Sensitive) Encrypted environment variables for Pages Functions.
```
Comment (atsushii) [inline docs/resources/pages_project.md:177]: this part is also generated by the `make docs` cmd.  Perhaps it is necessary too. so I just leave this diff.

Comment (jacobbednarz) [reply docs/resources/pages_project.md]: this was due to your fork not having the latest commits. i updated the branch with the latest and it is now only showing those changes for this PR.

**Comment by jacobbednarz** [comment]
thanks for this one; it looks great!

**Pull request merged**